specification/attestation/data-plane/Microsoft.Attestation/stable/2020-10-01/attestation.json

```diff
@@ -0,0 +1,1084 @@
+          "$ref": "#/definitions/JsonWebToken"
+        }
+      }
+    },
+    "PolicyCertificatesResult": {
+      "type": "object",
+      "description": "The result of a call to retrieve policy certificates.",
+      "properties": {
+        "x-ms-policy-certificates": {
+          "description": "SHA256 Hash of the binary representation certificate which was added or removed",
+          "$ref": "../../../../../common-types/rfcs/rfc7517.json#/definitions/JSONWebKeySet",
+          "x-ms-client-name": "PolicyCertificates"
+        }
+      }
+    },
+    "PolicyCertificatesModifyResponse": {
+      "type": "object",
+      "description": "The response to an attestation policy management API",
+      "properties": {
+        "token": {
+          "description": "An RFC7519 JSON Web Token structure whose body is a PolicyCertificatesModificationResult object.",
+          "$ref": "#/definitions/JsonWebToken"
+        }
+      }
+    },
+    "PolicyCertificatesModificationResult": {
+      "type": "object",
+      "description": "The result of a policy certificate modification",
+      "properties": {
+        "x-ms-certificate-thumbprint": {
+          "description": "Hex encoded SHA1 Hash of the binary representation certificate which was added or removed",
+          "type": "string",
+          "x-ms-client-name": "CertificateThumbprint"
+        },
+        "x-ms-policycertificates-result": {
+          "description": "The result of the operation",
+          "type": "string",
+          "x-ms-client-name": "CertificateResolution",
+          "enum": [
+            "IsPresent",
+            "IsAbsent"
+          ],
+          "x-ms-enum": {
+            "name": "CertificateModification",
+            "modelAsString": true,
+            "values": [
+              {
+                "value": "IsPresent",
+                "description": "After the operation was performed, the certificate is in the set of certificates."
+              },
+              {
+                "value": "IsAbsent",
+                "description": "After the operation was performed, the certificate is no longer present in the set of certificates."
+              }
+            ]
+          }
+        }
+      }
+    },
+    "StoredAttestationPolicy": {
+      "type": "object",
+      "properties": {
+        "AttestationPolicy": {
+          "description": "Policy text to set as a sequence of UTF-8 encoded octets.",
+          "type": "string",
+          "format": "base64url"
+        }
+      }
+    },
+    "PolicyResponse": {
+      "type": "object",
+      "description": "The response to an attestation policy operation",
+      "properties": {
+        "token": {
+          "description": "An RFC7519 JSON Web Token structure whose body is an PolicyResult object.",
+          "$ref": "#/definitions/JsonWebToken"
+        }
+      }
+    },
+    "PolicyResult": {
```

Comment:
Why this definition is not being referred to anywhere? cc @LarryOsterman

Reply:
The *Result types represent the body of JWT objects. The *Response types are the actual response from the service, I wanted to have the swagger express ALL of the types returned by the service (and this becomes incredibly helpful for the statically typed languages because they can return a defined type to the caller).

Reply:
I see. So you do not expect the code generator to use that type directly to type those JWT objects and the SDK writer will have to do that manually, right?

Reply:
Actually I hope that code generation will create the models for these types, even if they're unreachable. See https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/attestation/Azure.Security.Attestation/src/autorest.md where we force the generation of the models, so we can return them to the caller. I'm hoping to do the same thing for JS and Python.

Reply:
Yeah the models should be generated of course. My question was about using them.

Reply:
Here's an example of how the models are used in .Net:         
```
public virtual AttestationResponse<AttestationResult> AttestSgxEnclave(ReadOnlyMemory<byte> quote, BinaryData initTimeData, bool initTimeDataIsObject, BinaryData runTimeData, bool runTimeDataIsObject, CancellationToken cancellationToken = default)
```
Basically AttestSgxEnclave returns an AttestationResponse, an AttestationResponse includes the JWT and a Value property which returns an object of type AttestationResult. That allows clients of the AttestSgxEnclave API to directly access the body of the token as a structured object instead of as JSON.